libpod/oci_attach_linux.go

```diff
@@ -152,7 +152,7 @@ func (c *Container) attachToExec(streams *AttachStreams, keys string, resize <-c
 func processDetachKeys(keys string) ([]byte, error) {
 	// Check the validity of the provided keys first
 	if len(keys) == 0 {
-		keys = define.DefaultDetachKeys
+		return []byte{}, nil
```

Comment:
I don't know if this is safe for old containers - we might be disabling detach for every container created prior to this

Reply:
Yes, although I don't know a way around this.  Is the detach-key stored somewhere?  It does not get shown by podman inspect

Reply:
Actually after looking at the code, we don't save the detach keys anywhere.  So every podman start|exec|attach will use the default keys unless overridden.  We don't record the any of the following, as far as I can see
```type AttachValues struct {
	PodmanCommand
	DetachKeys string
	Latest     bool
	NoStdin    bool
	SigProxy   bool
}
```
This makes the 
podman create options ignored, I believe, because `podman start` will not use them.
Also `podman run/start/create` options, will not be used by `podman exec`, which matches what we have now, and I don't think it should block getting this in.  But we could discuss whether or not these options should be recorded.